P202.Training.Test/RoleServiceTest.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Moq;
+using P202.Training.Data.Entities;
+using P202.Training.Data.Repositories;
+using P202.Training.Domain;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace P202.Training.Test
+{
+    public class RoleServiceTest
+    {
+        [Fact]
+        public void RoleService_CreateData()
+        {
+            var newRole = new Role();
+            var mockpersonRepository = new Mock<IRoleRepository>();
```

Comment:
why is it called `mockpersonRepository` if it's the mock of the `RoleRepository`? Perhaps you should rename this variable

Reply:
Yes, it is a mistake. I copy/paste an example. I'll rename this variable as you suggest